marklogic-data-hub-central/ui/src/components/explore/entity-type-display-settings-modal/entity-type-display-settings-modal.tsx

```diff
@@ -0,0 +1,198 @@
+import React, {useState, useEffect} from "react";
+import {Modal} from "react-bootstrap";
+import Select from "react-select";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faPencilAlt} from "@fortawesome/free-solid-svg-icons";
+import styles from "./entity-type-display-settings-modal.module.scss";
+import HCTable from "@components/common/hc-table/hc-table";
+import HCButton from "@components/common/hc-button/hc-button";
+import graphConfig from "../../../config/graph-vis.config";
+import HCTooltip from "@components/common/hc-tooltip/hc-tooltip";
+import {QuestionCircleFill} from "react-bootstrap-icons";
+import tooltipsConfig from "../../../config/explorer-tooltips.config";
+import entityIcon from "../../../assets/Entity-Services.png";
+
+type Props = {
+  isVisible: boolean;
+  toggleModal: (isVisible: boolean) => void;
+  hubCentralConfig: any;
+  entityDefinitionsArray: any;
+};
+
+const {entityTypeDisplaySettings} = tooltipsConfig;
+
+const EntityTypeDisplaySettingsModal: React.FC<Props> = (props) => {
+  const [entityLabels, setEntityLabels] = useState({});
+  const [propertiesOnHover, setPropertiesOnHover] = useState({});
+  const [exploreSettingsData, setExploreSettingsData] = useState<any[]>([]);
+
+  useEffect(() => {
+    if (props.isVisible && props.hubCentralConfig && props.hubCentralConfig?.modeling?.entities) {
+      let settingsData:any = Object.keys(props.hubCentralConfig?.modeling?.entities).map(entityType => {
+        return {
+          entityType: entityType,
+          color: props.hubCentralConfig?.modeling?.entities[entityType]?.color ? props.hubCentralConfig?.modeling?.entities[entityType]?.color : "grey",
```

Comment:
Can we use the default icon and color here? I don't know if we have a utils or a file with the default (It would be nice to have one so we can change it in one place if we need to in the future)
Default icon: "FaShapes"
Default color: "#EEEFF1"

Reply:
@yubiselmarklogic - Let's check if we already have these default values defined in graph-vis.config (under config)  which we can reuse or else you can store them in explore.config.tsx file under config.